examples/tcp-retransmit.yaml

```diff
@@ -0,0 +1,56 @@
+metrics:
+    counters:
+      - name: tcp_retransmit_ipv4_count
```

Comment:
Counters in prometheus should end with `_total`:

* https://prometheus.io/docs/practices/naming/

My suggestion is `tcp_retransmit_ipv4_packets_total`.